check/request.go

```diff
@@ -0,0 +1,10 @@
+package check
+
+type VersionCheckRequest struct {
+	Product string
+	Version string
+	Arch    string
+	OS      string
+	DB      string
```

Comment:
Run `golint` on it. It might have different opinion about case here.

Reply:
Will do.